src/main/java/kdt/prgrms/kazedon/everevent/controller/UserController.java

```diff
@@ -159,9 +164,9 @@
     return ResponseEntity.noContent().build();
   }
 
-  public boolean isAuthenticated() {
+  public boolean isAnonymous() {
     Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
-    return (authentication != null) && !(authentication instanceof AnonymousAuthenticationToken);
+    return (authentication == null) || authentication instanceof AnonymousAuthenticationToken;
```

Comment:
괄호도 일관성있게 붙이면 좋을 것 같아요!ㅎㅎ

```suggestion
    return (authentication == null) || (authentication instanceof AnonymousAuthenticationToken);
```